test/characters-test.coffee

```diff
@@ -1,206 +0,0 @@
-chai      = require('chai')
```

Comment:
this is the test suite for y-text. If you want to use it in the characters, you should inherit (coffeescript *extend*) from it. If you `require "../../yjs/test/text-test"` the same tests will be executed. Furthermore, it contains lots of code that you only need to test once (because it actually only tests the ListManager - like the double late joins, observer tests, or "many operations, concurrently (random)").